apps/emqx_rule_engine/src/emqx_rule_events.erl

```diff
@@ -257,7 +257,7 @@ eventmsg_publish(
             flags => Flags,
             pub_props => printable_maps(emqx_message:get_header(properties, Message, #{})),
             %% the column 'headers' will be removed in the next major release
-            headers => printable_maps(Headers),
+            headers => printable_headers(Headers),
```

Comment:
I really want to remove the `header` field in 5.0. We should always get properties from the `pub_props` field.